src/main/java/Duke.java

```diff
@@ -1,10 +1,150 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println("    Hello from\n" + logo);
+        System.out.println("    _________________________________________");
+
+        System.out.println("    Hello! I'm Duke");
+        System.out.println("    What can I do for you?");
+        System.out.println("    _________________________________________");
+        System.out.println("     ");
+
+        //Read in input from user
+        String inputString;
+        Scanner in;
+
+        //Set up list to store user inputs
+        Todo[] tasks = new Todo[100];
+        int counter = 0;
+
+        //setup of exit flag
+        boolean exit = false;
```

Comment:
Perhaps, you might want to rename this boolean variable or add comments to describe what flag is meant to exit.
Maybe an alternative name like "boolean isByeCommand" might be more clearer to a reader.